src/plugins/dev.ts

```diff
@@ -37,7 +38,12 @@ export function DevPlugin(ctx: PWAPluginContext): Plugin {
 
         return html.replace(
           '</body>',
-            `<script type="module" src="${options.base}${DEV_SW_VIRTUAL}"></script></body>`,
+            `
+<script type="module">
```

Comment:
should we check CORS here?

Reply:
We could leave it for now, and bring it back if there is really a valid case.